chipflow_lib/platforms/silicon.py

```diff
@@ -8,7 +8,7 @@
 from amaranth import *
 from amaranth.back import rtlil
 from amaranth.hdl import Fragment
-from amaranth.lib.io import Pin
+from amaranth.lib.io import Direction, Buffer
 from amaranth.hdl._ir import PortDirection
```

Comment:
I think this private import can now be removed.

Reply:
Unfortunately not, `PortDirection` is used below (before calling `fragment.prepare()`).